<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaClasse</class>
 <widget class="QMainWindow" name="MaClasse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaClasse</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/nfond/fond.jpg);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/cartes/photos_des_cartes/interface.jpg);</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>340</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="deckLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>700</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="joeurLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>711</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="IALayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="labelCount">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>510</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/cartes/photos_des_cartes/BOTON.jpg);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="centralLayout"/>
   </widget>
   <widget class="QWidget" name="mywidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1262</width>
      <height>912</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/cartes/photos_des_cartes/interface.jpg);</string>
    </property>
    <widget class="QPushButton" name="PlayButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>510</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>24</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 0);
background-image: url(:/photos_des_cartes/play.jpg);
font: 900 24pt &quot;Segoe UI&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string>PLAY</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>210</y>
       <width>411</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(191, 28, 58);
border-color: rgb(255, 255, 255);


font: 24pt &quot;Segoe UI&quot;;
background-image: url(:/photos_des_cartes/mytitle.jpg);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;JEU DE CARTES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
